resources/otoole.yaml

```diff
@@ -42,7 +42,7 @@ CapitalCostStorage:
     indices: [REGION,STORAGE,YEAR]
     type: param
     dtype: float
-    default: 2000
+    default: 0.001
```

Comment:
Was this change deliberate? Doesn't matter either way as storage costs are defined within the workflow was just wondering

Reply:
Was deliberate, and I just aligned it with powerplant capital cost default values